web_src/js/features/repo-issue.js

```diff
@@ -287,6 +287,33 @@ export function initRepoPullRequestMergeInstruction() {
   });
 }
 
+export function initRepoPullRequestAllowEditsFromMaintainers() {
+  $('#allow-edits-from-maintainers input').on('change', function () {
+    const $label = $('#allow-edits-from-maintainers label');
+    let url = $label.data('url');
+    if (this.checked) { // allow edits
+      url = `${url}/allow_edits`;
+    } else { // disallow edits
+      url = `${url}/disallow_edits`;
```

Comment:
Can we use single URL: `/update_allow_edits`?

Reply:
The problem might be if the server is not reachable and an error occures: at least Firefox caches the state of this checkbox and ignores the values given from the server, so there is no chance to find out in which state the PR currently is. With separate URLs, you can
1. Post manually to them
2. If I click the checkbox even with a cached state, the action will be triggered that the checkbox currently has - independent from the real state on the server

Reply:
or toggle_allow_edits? But the current solution is accepted by me.

You can disable the checkbox when updating. And of course, you should consider timeout.